_posts/EternalFootman/2016-01-14-turtles.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+author: EternalFootman
+title: "Kit's Turtles"
+---
+
+Here are Tina's desert adventures.
+
+<iframe src="https://trinket.io/embed/python/6e3d7d7d02?start=result" width="100%" height="356" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
```

Comment:
This URL sends me to the original.  Did you get the URL via the Share link on the trinket? Otherwise what I see will not be what you see.